curriculum/js2/5.js

```diff
@@ -4,8 +4,16 @@
  * @returns {number}
  */
 
-const solution = (a) => {
-  return 0;
+const solution = (a, idx = 1, largest = a[0]) => {
+  if (idx >= a.length) {
+    return a.length === 0 ? 0 : largest;
```

Comment:
same comment as my last. this ternary operator is unnecessary to make the code work, no?

Reply:
The previous one did not require a ternary, but I believe this one does need it to handle the empty array case. In the prior example, sum was initialized to zero. In this example a[0] will evaluate to undefined and we want to return zero in that case.